sdk/spring/azure-spring-integration-servicebus/src/main/java/com/azure/spring/integration/servicebus/factory/DefaultServiceBusQueueClientFactory.java

```diff
@@ -36,6 +37,11 @@ public DefaultServiceBusQueueClientFactory(String connectionString) {
 
     }
 
+    public DefaultServiceBusQueueClientFactory(String connectionString, AmqpTransportType amqpTransportType) {
```

Comment:
How about making the single para constructor do nothing but call the multi-para one with a default AmqpTransportType value like AMQP?